src/org/labkey/test/tests/GroupTest.java

```diff
@@ -72,12 +72,13 @@ protected String getProject2Name()
     @Override
     protected void doCleanup(boolean afterTest) throws TestTimeoutException
     {
-        _permissionsHelper.deleteGroup(COMPOUND_GROUP);
-        _permissionsHelper.deleteGroup(SIMPLE_GROUP);
-        _permissionsHelper.deleteGroup(BAD_GROUP);
-        _permissionsHelper.deleteGroup(CHILD_GROUP);
-        _permissionsHelper.deleteGroup(SITE_USER_GROUP);
-        _permissionsHelper.deleteGroup(API_SITE_GROUP);
+        ApiPermissionsHelper permissionsHelper = new ApiPermissionsHelper(this);
```

Comment:
Since this test is focused on checking permissions, we want the rest of the test to go through the UI. We just want to use the API here for quick cleanup.